frontstage/error_handlers.py

```diff
@@ -65,3 +65,9 @@ def server_error(error):
 def eq_error(error):
     logger.error('Failed to generate EQ URL', error=error.message, url=request.url, status_code=500)
     return render_template('errors/500-error.html'), 500
+
+
+@app.errorhandler(IncorrectAccountAccessError)
+def secure_message_forbidden_error(error):
+    logger.error('Attempt to access secure message without correct session permission')
```

Comment:
I think this should be info level rather than error - it's a user error rather than one we need to know about.

Reply:
Also, could you add more context to the error message please (userid, thread_id, whatever you have), as the message isn't much help on its own